libvast/vast/chunk.hpp

```diff
@@ -27,7 +27,7 @@
 
 namespace vast {
 
-/// A contiguous block of memory; essentially a span of bytes with a deleter.
+/// A contiguous block of memory.
```

Comment:
You could help the reader a little bit more by explaining the copy/reference semantics. For example, *"A reference-counted contiguous block of memory. A chunk supports custom deleters for custom deallocations when the last instance goes out of scope."*

Reply:
```suggestion
/// A reference-counted contiguous block of memory. A chunk supports custom
/// deleters for custom deallocations when the last instance goes out of scope.
```